examples/jsm/nodes/lighting/AnalyticLightNode.js

```diff
@@ -73,11 +73,24 @@ class AnalyticLightNode extends LightingNode {
 				.and( shadowCoord.y.lessThanEqual( 1 ) )
 				.and( shadowCoord.z.lessThanEqual( 1 ) );
 
-			shadowCoord = vec3(
-				shadowCoord.x,
-				shadowCoord.y.oneMinus(), // WebGPU: Flip Y
-				shadowCoord.z.add( bias ).mul( 2 ).sub( 1 ) // WebGPU: Convertion [ 0, 1 ] to [ - 1, 1 ]
-			);
+
+			if ( builder.isAdjustShadowCoord() ) {
```

Comment:
I think it would be better to use `builder.renderer.coordinateSystem === WebGPUCoordinateSystem`

Reply:
Yes, much better. Fixed